### PR TITLE
1122

### PR DESCRIPTION
**Pull request by chenyan19209219**

### CODE REVIEW

File: src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java [hunk 1]
```diff
@@ -13,6 +13,10 @@ public String home() {
         return HELLO_WEB;
     }
     
+
+    @RequestMapping("/hello2")
```
Comment (chenyan19209219) [inline src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java]: 123

Comment (chenyan19209219) [reply src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java]: werwereqer

File: src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java [hunk 2]
```diff
@@ -13,6 +13,10 @@ public String home() {
...
+    public String home2() {
+
     @RequestMapping("/home3")
```
Comment (chenyan19209219) [inline src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java]: wwwwww

File: src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java (hunk 1 above)
Comment (katerina19209219) [reply src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java]: ghj,

Comment (katerina19209219) [reply src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java]: dfghj

Comment (katerina19209219) [reply src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java]: cvbnm,

File: src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java [hunk 3]
```diff
@@ -13,6 +13,10 @@ public String home() {
...
     public String home3() {
```
Comment (katerina19209219) [inline src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java]: zxcvbnm,

File: src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java (hunk 1 above)
Comment (chenyan19209219) [reply src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java]: cvbnm

**Comment by XingKaihang** [comment]
common comment

File: src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java (hunk 1 above)
Comment (XingKaihang) [inline src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java]: aaa

File: src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java [hunk 4]
```diff
@@ -13,6 +13,10 @@ public String home() {
         return HELLO_WEB;
     }
     
+
+    @RequestMapping("/hello2")
+    public String home2() {
```
Comment (XingKaihang) [inline src/main/java/com/aliexpress/spring/boot/demo/controller/HomeController.java]: ffff

**Pull request merged**